modules/op-resolver-git-override-scm.adoc

```diff
@@ -0,0 +1,74 @@
+// This module is included in the following assembly:
+//
+// openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+
+:_mod-docs-content-type: REFERENCE
+= Specifying a remote pipeline or task by using the Git resolver with the authenticated SCM API overriding the Git resolver configuration
+
+When creating a pipeline run, you can specify a remote pipeline from a Git repository by using the authenticated SCM API. When creating a pipeline or a task run, you can specify a remote task from a Git repository. You can override the initial configuration settings in specific pipeline runs or tasks to customize the behavior according to different use cases.
+
+The following example task run references a remote task from a Git repository that overrides the previous resolver configuration:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: git-task-reference-demo
+spec:
+  taskRef:
+    resolver: git
+    params:
+    - name: org
+      value: tektoncd
+    - name: repo
+      value: catalog
+    - name: revision
+      value: main
+    - name: pathInRepo
+      value: task/git-clone/0.6/git-clone.yaml
+    - name: token
+      value: my-secret-token
+    - name: tokenKey
+      value: token
+    - name: scmType
+      value: github
+    - name: serverURL
+      value: https://ghe.mycompany.com
+----
+.Supported parameters to override the Git resolver
+|===
+| Parameter | Description | Example value
+
+| `org`
+| The organization for the repository.
+| `tektoncd`
+
+| repo
+| The repository name.
+| catalog
+
+| revision
+| The Git revision in the repository. You can specify a branch name, a tag name, or a commit SHA hash.
+| main
+
+| pathInRepo
+| The path name of the YAML file in the repository.
+| task/git-clone/0.6/git-clone.yaml
+
+| token
+| The secret name used for authentication.
+| my-secret-token
+
+| tokenKey
+| The key name for the token.
+| token
+
+| scmType
+| The type of SCM (Source Control Management) system.
+| github
+
+| serverURL
+| The URL of the server hosting the repository.
+| `+https://ghe.mycompany.com+`
```

Comment:
I meant for the whole table to get the marked-up treatment (where appropriate), but not going to block merge on this. But consider for a follow-up.